bin/capture.zsh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env zsh
+#!/bin/zsh
```

Comment:
I know original source use `!#/bin/zsh`, but typical OS distribution(at least macOS) users using `/usr/local/bin/zsh`, so I fixed this line.
In macOS, default `/bin/zsh` is `5.2`(on Sierra dev), but I use `/usr/local/bin/zsh` which `5.3.1-dev-0`.
So, It will modify the original, but `/bin/zsh` is not a good choice.